schema/2.0.0/corpora_schema.md

```diff
@@ -101,9 +101,9 @@ The following ontology dependencies are *pinned* for this version of the schema.
 | [Cell Ontology] | CL | [cl.owl] : [2021-08-10]|
 | [Experimental Factor Ontology] | EFO | [efo.owl] : [2021-08-16 EFO 3.33.0]
 | [Human Ancestry Ontology] | HANCESTRO |[hancestro.owl] : [2021-01-04 (2.5)] |
-| [Human Developmental Stages] |  HsapDv | [hsapdv.owl] : [2016-07-06 (0.1)] |
+| [Human Developmental Stages] |  HsapDv | [hsapdv.owl] : 2020-03-10 (1.2) |
 | [Mondo Disease Ontology] | MONDO |[mondo.owl] : [2021-08-11] |
-| [Mouse Developmental Stages]| MmusDv |  [mmusdv.owl] : [2016-07-06 (0.1)] |
+| [Mouse Developmental Stages]| MmusDv |  [mmusdv.owl] : 2020-03-10 (1.2) |
```

Comment:
I'm curious where you got the `1.2`s from

Reply:
Doh. I made a bad assumption that obo format version was the ontology version. Will correct.

Reply:
Corrected in e80f2f4cd28372318a296cada9887ed50639bfc3